Button/Button.module.less

```diff
@@ -42,14 +42,24 @@
 		}
 
 		.client {
-			padding: @agate-button-border-width; // We match the amount removed by the line-height above, so the text doesn't overlap the border
+			padding: @agate-button-client-padding;
 			color: @agate-button-color;
 
 			.icon {
 				.margin-start-end(@agate-button-icon-margin-start, @agate-button-icon-margin-end);
 			}
 		}
 
+		.disabled({
+			.bg {
```

Comment:
For disabled style, opacity 0.6 was applied before.
This impede consistency depending on the background color.
So, GUI person said that some fixed color will apply for disabled. Could you remove the opacity style that was previously applied only to silicon?
CC : @adrian-cocoara-lgp @stanca-pop-lgp @paul-beldean-lgp @0x64 @MikyungKim

Reply:
Alright. This was actually the cause of why I used 50% opacity for colors instead of the 30% from the GUI guide.

Reply:
@jeonghee27  Actually, is it ok to set @agate-disabled-opacity to 1? That will affect all components. Or maybe I misunderstood which previosuly opacity of 0.6 you meant.

Reply:
Oh, I didn't say it clearly. First of all, I wanted to remove it only from the silicone button.
And, it still looks opacity applied. It doesn't look like #444444.
![image](https://user-images.githubusercontent.com/34676178/110875526-239f5e00-8319-11eb-8e64-ef9fec3a2edd.png)

Reply:
I don't know what happened, but apparently, the @agate-disabled-opacity: 1 setting was not pushed. I fixed now the opacity only for Silicon Button